partner_contact_nutrition/models/res_partner.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Ursa Information Systems <http://ursainfosystems.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+from openerp import fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    caloric_intake = fields.Float("Calories")
+    caloric_intake_uom = fields.Many2one("product.uom", "Calories UoM")
```

Comment:
@lasley I have done it in the view. I haven't found how to put the xml ID in the domain on the model.